src/main/java/model/TransactionDto.java

```diff
@@ -0,0 +1,57 @@
+package model;
```

Comment:
check the packages of your classes, most of them are not in the `core.basesyntax` but they should be there